source/RialtoGStreamerMSEAudioSink.cpp

```diff
@@ -45,8 +45,21 @@ static GstStateChangeReturn rialto_mse_audio_sink_change_state(GstElement *eleme
     {
     case GST_STATE_CHANGE_READY_TO_PAUSED:
     {
+        // Attach the media player client to media player manager
+        GstObject parentObject = rialto_mse_base_get_oldest_gst_bin_parent(element);
```

Comment:
I haven't checked if code which I proposed compiles. I think that parentObject should be a pointer.

Reply:
Yes it compiles now :D